compiler+runtime/test/bash/main-exit/pass-test

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bb
+
+(ns jank-test-main-exit
+  (:require [babashka.process :as proc]
+            [clojure.string :as str]
+            [clojure.test :as t :refer [deftest run-test is]]))
+
+(def this-ns (ns-name *ns*))
+
+(deftest jank-run-main-successful-exit
+  (let [{:keys [err out exit]}
+        (proc/shell {:out :string :err :string :continue true}
```

Comment:
I would break all of these map entries onto their own lines, too. There's nothing wrong with more lines!